File: sdks/go/pkg/models/shared/metadatatemplates.go
package shared

// MetadataTemplates
// The part of an API response that describes marker
// based pagination
type MetadataTemplates struct {
	Entries    []MetadataTemplate `json:"entries,omitempty"`
	Limit      *int64             `json:"limit,omitempty"`
	NextMarker *int64             `json:"next_marker,omitempty"`
	PrevMarker *int64             `json:"prev_marker,omitempty"`
}

File: python-sdk/src/boxsdk/models/shared/metadatatemplates.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from boxsdk import utils
from ..shared import metadatatemplate as shared_metadatatemplate


@dataclass_json
@dataclasses.dataclass
class MetadataTemplates:
    r"""MetadataTemplates
    The part of an API response that describes marker
    based pagination
    """
    
    entries: Optional[list[shared_metadatatemplate.MetadataTemplate]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('entries') }})
    limit: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('limit') }})
    next_marker: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('next_marker') }})
    prev_marker: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('prev_marker') }})
    
